Libraries/Text/TextInput/RCTBaseTextInputViewManager.m

```diff
@@ -46,6 +46,7 @@ @implementation RCTBaseTextInputViewManager
 RCT_REMAP_VIEW_PROPERTY(placeholderTextColor, backedTextInputView.placeholderColor, UIColor)
 RCT_REMAP_NOT_OSX_VIEW_PROPERTY(returnKeyType, backedTextInputView.returnKeyType, UIReturnKeyType) // TODO(macOS ISS#2323203)
 RCT_REMAP_NOT_OSX_VIEW_PROPERTY(secureTextEntry, backedTextInputView.secureTextEntry, BOOL) // TODO(macOS ISS#2323203)
+RCT_REMAP_OSX_VIEW_PROPERTY(secureTextEntry, useSecureTextField, BOOL) // TODO(macOS ISS#2323203)
```

Comment:
Since a `useSecureTextField` property wasn't added to `RCTBaseTextInputView`, would this crash if set on a `RCTMultilineTextInputView`?

Reply:
Indeed: it throws an '... unrecognized selected send to instance' exception.